schema

```diff
@@ -0,0 +1,33 @@
+collection users
+{
+	name : "mayuri",
+	surname: "ardad",
+	login_credentials:{
+						username: "mayuri"
+						password: "something"
+
+						},
+	role: {
+			position: "admin",
+			people_assigned: [....],
+			},
+	posts: [.....]
```

Comment:
user - posts is a 1-M relationship, the user shouldn't have the post_ids as an array, but  instead the user id should be in my posts collection